tests/integration_tests/testsuite/ovc_tests/a_basic/accounts_tests.py

```diff
@@ -70,11 +74,17 @@ def test002_create_account_with_correct_params(self):
         CU_C = randint(15, 30)
         CU_I = randint(15, 30)
         CU_M = randint(15, 30)
-        self.accounts[self.acc1] = {'openvcloud': self.openvcloud, 'maxMemoryCapacity': CU_M,
+        self.accounts[self.acc1] = {'openvcloud': self.openvcloud,
+                                    'name': self.acc1_name,
+                                    'maxMemoryCapacity': CU_M,
                                     'maxCPUCapacity': CU_C, 'maxVDiskCapacity': CU_D,
-                                    'maxNumPublicIP': CU_I }
+                                    'maxNumPublicIP': CU_I,
+                                    'users': OrderedDict([('name', self.vdcuser),
```

Comment:
why users section is back here?

Reply:
I think it's old version , I will delete it